projects/ccf-rui/src/app/core/store/registration/registration.state.ts

```diff
@@ -104,12 +129,25 @@ export class RegistrationState extends NgxsImmutableDataRepository<RegistrationS
     this.ctx.patchState({ useRegistrationCallback: use });
   }
 
+  /**
+   * Set's whether or not we should display the user's registration errors
+   * @param displayErrors the value to set it to
+   */
+  @DataAction()
+  setDisplayErrors(displayErrors: boolean): void {
+    this.ctx.patchState({ displayErrors });
+  }
+
   /**
    * Registers or downloads json data.
    *
    * @param [useCallback] Explicit override selecting the register/download action
    */
   register(useCallback?: boolean): void {
+    if (!this.valid$) {
```

Comment:
This is an observable so this if will always be false. Should be `!this.isValid(this.page.snapshot, this.model.snapshot)`

Reply:
good catch.  I'll get that changed out